### PR TITLE
Fixes #20310 - handle missing option

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: lib/foreman_tasks/dynflow/daemon.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def run(foreman_root = Dir.pwd, options = {})
 
       ForemanTasks.dynflow.executor!
 
-      if options[:memory_limit] > 0
+      if options[:memory_limit] && options[:memory_limit] > 0
```
Comment (ehelms) [inline lib/foreman_tasks/dynflow/daemon.rb]: You repeated the same thing here :)

Comment (iNecas) [inline lib/foreman_tasks/dynflow/daemon.rb]: What a nitpicker! :D Updated

**Comment by dLobatog** [comment]
Not sure if this makes sense to merge now - it's incorporated in https://github.com/Dynflow/dynflow/pull/236

**Comment by iNecas** [comment]
Depends if @ehelms needs this for nightlies only, or if this should be backported do 1.15? If nightlies, I agree with having this fixed in dynflow as enough

**Comment by ehelms** [comment]
Currently, I have only seen this break in nightlies.

**Comment by ehelms** [comment]
However, this does upgrade the code to be more robust to ensure tight checking no matter what fix is in Dynflow (unless this entire chunk of code would be removed).

**Comment by iNecas** [comment]
In dynflow is the very same test, closing for now

**Pull request closed without merging**